src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -80,48 +80,57 @@ internal sealed class AnalyzerDependencyCheckingService
 
             var conflicts = results.Conflicts;
             var missingDependencies = results.MissingDependencies;
+            var projects = _workspace.DeferredState.ProjectTracker.ImmutableProjects;
```

Comment:
@heejaechang Updated `HostDiagnosticUpdateSource.UpdateDiagnosticsForProject` to only add project diagnostics if the project exists in the current solution.